code/modules/clothing/glasses/hud.dm

```diff
@@ -8,9 +8,9 @@
 
 
 /obj/item/clothing/glasses/hud/equipped(mob/living/carbon/human/user, slot)
-	..()
+	. = ..()
 	if(slot != ITEM_SLOT_EYES)
-		return
+		return FALSE
```

Comment:
huh??

Reply:
I am just using this to check if the item was equipped correctly.